test/management/management-client.tests.js

```diff
@@ -25,7 +26,10 @@ describe('ManagementClient', function() {
   var withTokenProviderConfig = {
     clientId: 'clientId',
     clientSecret: 'clientSecret',
-    domain: 'auth0-node-sdk.auth0.com'
+    domain: 'auth0-node-sdk.auth0.com',
+    tokenProvider: {
```

Comment:
The token cache bleeds between test cases causing other tests to fail if I have it enabled here.  I suggest in the future we have a method that clears the token cache between each test case so this doesn't happen in the future.